tests/files/datasets/gebieden.json

```diff
@@ -45,12 +43,12 @@
           },
           "code": {
             "type": "string",
-            "description": "Officiële code van het object."
+            "description": "Offici\u00eble code van het object."
           },
           "beginGeldigheid": {
             "type": "string",
             "format": "date",
-            "description": "De datum waarop het object is gecreëerd."
+            "description": "De datum waarop het object is gecre\u00eberd."
```

Comment:
Waar komen al die escape chars vandaan? JSON ondersteund unicode

Reply:
hmm heb t ff snel met een python script gedaan, zal wel een ascii decoding ergens tussen in perongeluk zijn geweest